dynamic_rest/serializers.py

```diff
@@ -393,6 +393,14 @@ def get_fields(self):
         ro_fields = getattr(self.Meta, 'read_only_fields', [])
         self.flag_fields(serializer_fields, ro_fields, 'read_only', True)
 
+        pw_fields = getattr(self.Meta, 'preserve_whitespace', [])
```

Comment:
I'd prefer to keep the convention of ending these meta fields with `_fields` (just `preserved_whitespace` could look like it should be a boolean flag), but not sure what a non-awkward name would be. Maybe `whitespace_preserved_fields`?